anvil-rstudio-bioconductor/Dockerfile

```diff
@@ -0,0 +1,124 @@
+FROM anvil-rstudio-base:0.0.2
```

Comment:
This needs to be
```
FROM us.gcr.io/anvil-gcr-public/anvil-rstudio-base:0.0.2
```
(I got a "repository not found error otherwise")

Reply:
Yes, so, I didn't push anything up to us.gcr.io/anvil-gcr-public because I couldn't actually launch these images on the terra platform to see and RStudio image. 

But I can push an image to `us.gcr.io/anvil-gcr-public/anvil-rstudio-base:0.0.2` so we can run it this way.

Reply:
Nvm, It seems you just pushed to the anvil-gcr-public directory. Thank you :D

Reply:
Yep :)